Automattic/wp-calypso
# Reader: launch refreshed full post to everyone

The new Reader full post view is currently being served to 50% of users in an A/B test.

This PR removes the A/B test and serves the new full post view to everyone.

### To test

Open your Following stream at http://calypso.localhost:3000/ and open a selection of posts in full post view. Ensure you see the refreshed full post view and everything works as expected:

<img width="1087" alt="screen shot 2016-10-27 at 16 05 11" src="https://cloud.githubusercontent.com/assets/17325/19753176/30944788-9c5f-11e6-87fd-17a168a7b1bf.png">
